client/src/components/pages/admin/DonutChart.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {render} from 'react-dom';
+import ReactDOM from 'react-dom';
+import ReactNativeSlider from "react-html5-slider";
+import ReactFauxDOM from 'react-faux-dom';
```

Comment:
Please use [this](http://www.reactd3.org/) to generate charts with D3 and react instead of rolling your own.